api/client/webclient/webclient.go

```diff
@@ -328,6 +330,14 @@ type ProxySettings struct {
 	TLSRoutingEnabled bool `json:"tls_routing_enabled"`
 }
 
+// AutoUpdateSettings contains information about the auto update requirements.
+type AutoUpdateSettings struct {
+	// ToolsVersion defines the version of {tsh, tctl} for client auto update.
+	ToolsVersion string `json:"tools_version"`
+	// ToolsMode defines mode client auto update feature `enabled|disabled`.
+	ToolsMode string `json:"tools_mode"`
```

Comment:
@vapopov I think this should be `tools_auto_update`?

https://github.com/gravitational/teleport/blob/master/api/client/webclient/webclient.go#L363